lib/msal-common/rollup.config.js

```diff
@@ -39,6 +41,38 @@ export default [
             })
         ]
     },
+    {
+        // for es build with hashed logs
+        input: ["src/index-browser.ts", "src/index-node.ts", "src/index.ts"],
+        output: {
+            dir: "dist-browser",
```

Comment:
If this is supposed to replace the existing browser output we should remove index-browser from the inputs to the original es build step